llama_stack/providers/adapters/inference/bedrock/bedrock.py

```diff
@@ -290,23 +290,130 @@ async def chat_completion(
         messages: List[Message],
         sampling_params: Optional[SamplingParams] = SamplingParams(),
         response_format: Optional[ResponseFormat] = None,
-        # zero-shot tool definitions as input to the model
         tools: Optional[List[ToolDefinition]] = None,
         tool_choice: Optional[ToolChoice] = ToolChoice.auto,
         tool_prompt_format: Optional[ToolPromptFormat] = ToolPromptFormat.json,
         stream: Optional[bool] = False,
         logprobs: Optional[LogProbConfig] = None,
-    ) -> (
-        AsyncGenerator
-    ):  # Union[ChatCompletionResponse, ChatCompletionResponseStreamChunk]:
-        bedrock_model = self.map_to_provider_model(model)
+    ) -> AsyncGenerator:
```

Comment:
this should be now `Union[ChatCompletionResponse, AsyncIterator[ChatCompletionResponseStreamChunk]]` -- see the corresponding type in `apis/inference/inference.py`

Reply:
done.